src/components/CartItems.jsx

```diff
@@ -0,0 +1,102 @@
+import ReactStarsRating from "react-awesome-stars-rating";
+import { addToCart, removeFromCart } from "../redux/actions";
+import { useNavigate } from "react-router-dom";
+import { useDispatch, useSelector } from "react-redux";
+import { Flex, Text, Button } from "@sparrowengg/twigs-react";
+import EmptyCartUI from "./EmptyCart";
+const CartItems = () => {
+  const cart = useSelector((state) => state.cart);
+  const dispatch = useDispatch();
+  const navigate = useNavigate();
+
+  return (
+    <>
+      {cart.length === 0 ? (
+        // <Flex css={{ width: "100%" }}>
+        <EmptyCartUI />
+      ) : (
+        // </Flex>
+        <Flex flexDirection="column" css={{ overflow: "scroll" }}>
+          <Text
+            css={{
+              marginBlock: "$15",
+              textAlign: "center",
+              fontSize: "$3xl",
```

Comment:
@arun-121 can you use Heading component instead?

Reply:
removed fontsize and used Heading component